lib/plugins/common/ccxt.js

```diff
@@ -0,0 +1,28 @@
+const _ = require('lodash/fp')
+
+const CRYPTO = {
+  bitstamp: ['BTC', 'ETH', 'LTC', 'BCH'],
+  itbit: ['BTC', 'ETH', 'LTC', 'BCH'],
+  kraken: ['BTC', 'ETH', 'LTC', 'DASH', 'ZEC', 'BCH'],
+  coinbase: ['BTC', 'ETH', 'LTC', 'BCH', 'ZEC', 'DASH']
+}
+
+const FIAT = {
+  bitstamp: ['USD', 'EUR'],
+  itbit: ['USD'],
+  kraken: ['USD', 'EUR']

```

Comment:
Kraken has a CAD market that some prefer.

But moreover, we should have an admin toggle, if not already planned, whereby one can choose which market to trade on.